packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -301,6 +300,7 @@ const Dropdown = React.forwardRef(
               isOpen={isOpen}
               label={label}
               full={full}
+              ref={ref}
```

Comment:
One last comment... add like `const inputRef = ref || useRef(null)` so if no ref is present will be handled to null, what do you think